### PR TITLE
Generate new rails app

### PR DESCRIPTION
**Pull request by thatbudakguy**
Includes options for esbuild and bootstrap.
Refs #1

### CODE REVIEW

**Comment by thatbudakguy** [comment]
yeah, I guess this is more a test/proof of concept than something we would actually merge. maybe we want to think about resolving both of those things in arclight core?

**Comment by marlo-longley** [comment]
@thatbudakguy yes, I am thinking updated to BS 5 in Arlight Core may be on deck.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "3.1.2"
```
Comment (cbeer) [inline Gemfile:4]:
```suggestion
```

Ruby versions in the Gemfile make it annoying to deploy.

**Comment by thatbudakguy** [comment]
closing in favor of #14; we can always switch strategies later

**Pull request closed without merging**